### PR TITLE
Download multi pdf bills in zip

### PR DESCRIPTION
**Pull request by andrelec1**
Add option for download multi pdf in zip.

bwj7
tijc
fx0z
rwoe
l8w7

### CODE REVIEW

**Comment by fappels** [comment]
Nice feature, but it would be more nice if it could be an option in the EDM area.

File: htdocs/compta/facture/list.php [hunk 1]
```diff
@@ -51,6 +52,41 @@
...
+	}
+
+	//Create dir for zip if not exist
+	$diroutputzip=$conf->facture->dir_output . '/FactBill';
+	if(dol_mkdir($diroutputzip) < 0){
+		die("Error mkdir;");
+	};
+
+	//Creating Zip File
+	$zip = new ZipArchive();
```
Comment (rdoursenaud) [inline htdocs/compta/facture/list.php:73]: This relies on the zip extension that may not be available.
You'd want to wrap it into a

``` php
if (class_exists('ZipArchive')) {
    […]
}
```

and either disable the functionality entirely and/or issue a message to the user.

**Comment by marcosgdf** [comment]
Agree with @fappels

**Comment by rvjpub** [comment]
nice good job

2014-12-21 20:27 GMT+01:00 Marcos García notifications@github.com:

**Comment by eldy** [comment]
The page to list invoice table is currently being merged with the feature to build a merged pdf. This means code of page is changed to introduce checkbox on each line to select invoice. We may reintroduce this feature once merge will be finished.

**Pull request closed without merging**